src/patternfly/components/Card/examples/Card.md

```diff
@@ -384,4 +416,6 @@ A card is a generic rectangular container that can be used to build other compon
 | `.pf-m-selected` | `.pf-c-card.pf-m-selectable` | Modifies a selectable card for the selected state. |
 | `.pf-m-flat` | `.pf-c-card` | Modifies the card to have a border instead of a shadow. `.pf-m-flat` is for use in layouts where cards are against a white background. |
 | `.pf-m-rounded` | `.pf-c-card` | Modifies the card to have rounded corners. |
+| `.pf-m-no-box-shadow` | `.pf-c-card` | Modifies the card to have rounded corners. |
 | `.pf-m-expanded` | `.pf-c-card` | Modifies the card for the expanded state. |
+| `.pf-m-toggle-right` | `.pf-c-card__header` | Modifies the card header toggle to be positioned at flex-end. |
```

Comment:
WDYT about this? `.pf-c-card.pf-m-expandable .pf-c-card__header` could work, too, though I think the description update is better.

```suggestion
| `.pf-m-toggle-right` | `.pf-c-card__header` | Modifies the expandable card header toggle to be positioned at flex-end. |
```